src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/dwpchallengevalidity/DwpChallengeValidityAboutToSubmitHandlerTest.java

```diff
@@ -90,9 +90,10 @@ public void givenDwpChallengeValidityEventWithExistingDwpDocuments_thenMoveDocsT
         PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
 
         assertNull(response.getData().getDwpChallengeValidityDocument());
+        assertEquals("existing.com", sscsCaseData.getDwpDocuments().get(0).getValue().getDocumentLink().getDocumentUrl());
         assertEquals(2, response.getData().getDwpDocuments().size());
-        assertEquals("challengeValidityLink", response.getData().getDwpDocuments().get(0).getValue().getDocumentLink().getDocumentUrl());
-        assertEquals(DwpDocumentType.DWP_CHALLENGE_VALIDITY.getValue(), response.getData().getDwpDocuments().get(0).getValue().getDocumentType());
+        assertEquals("challengeValidityLink", response.getData().getDwpDocuments().get(1).getValue().getDocumentLink().getDocumentUrl());
+        assertEquals(DwpDocumentType.DWP_CHALLENGE_VALIDITY.getValue(), response.getData().getDwpDocuments().get(1).getValue().getDocumentType());
```

Comment:
Is this list ordered or does the order matter, if not I would recommend using assertThat().extracting.contains() or hamcrest contains for the assertions as this can fail depending on the order of the list (had the problem previously)

Reply:
Yes the list is ordered and the ordering was changed so the current tests had to be updated as these tests were not added by Lillie I do not believe she should change them or mess around with them really. Seems like they were already there

Reply:
Tests were already there, they needed updated to accommodate the new document ordering